osimpipeline/task.py

```diff
@@ -96,29 +105,183 @@ def __init__(self, trial):
         super(TrialTask, self).__init__(trial.subject)
         self.trial = trial
 
+class SetupTask(TrialTask):
+    def __init__(self, tool, trial, 
+                 model_to_adjust=None, 
+                 adjusted_model=None,
+                 gait_cycles='concatenated'):
+        super(SetupTask, self).__init__(trial)
+        self.tool = tool
+        self.trial = trial
+        self.name = '%s_%s_setup' % (trial.id, self.tool)
+        self.tool_path = os.path.join(trial.results_exp_path, self.tool)
+        self.doit_path = self.study.config['doit_path']
```

Comment:
`self.doit_path` is just a string path to the directory where dodo.py sits. I found it convenient since I wanted to leave some files there (e.g. RRA/CMC actuators). I guess `self.dodo_path` could be more accurate, but I have, let's say, _aesthetic difficulties_ with that name.